docker/gitlab-ci/Dockerfile

```diff
@@ -60,7 +61,8 @@ RUN \
   apt-get update && apt-get install -y --force-yes google-cloud-sdk
 
 RUN \
-  apt-get install -y --force-yes dnsutils
+  curl -fsSL https://deb.nodesource.com/setup_12.x | sudo -E bash - && \
```

Comment:
I suggest working with nvm and having during the build install the proper node based upon `.nvmrc`. See example here: https://github.com/demisto/content-docs/blob/master/.circleci/config.yml#L29